### DISCUSSION TITLE
governance model?

### DISCUSSION THREAD

**Discussion by jacobweinstock** [tinkerbell/tink#504] [General]
What is our governance model? If we don't already have one defined, let's discuss and formalize one!
https://opensource.guide/leadership-and-governance/#what-are-some-of-the-common-governance-structures-for-open-source-projects

**Comment by jacobweinstock** [comment]
![image](https://user-images.githubusercontent.com/12081036/121096325-3659ef80-c7af-11eb-8a24-689cc6491cce.png)

these teams seem to be Equinix specific.  This is part of the governance that I think we need to make open and define a model for different roles. 
ref: https://opensource.guide/leadership-and-governance/#what-are-examples-of-formal-roles-used-in-open-source-projects

**Comment by markjacksonfishing** [comment]
A framework we can refer to for [Kubernetes](https://github.com/kubernetes/community/blob/master/governance.md) or [Jenkins](https://www.jenkins.io/project/governance/)

**Reply by jacobweinstock** [reply]
I like the "Project roles/stakeholders" in the Jenkins link you posted. Very clear and understandable.

**Reply by markjacksonfishing** [reply]
yeah, no ambiguity is what I really liked about it

**Comment by jacobweinstock** [comment]
I've put something concrete out there to get us going. https://github.com/tinkerbell/proposals/pull/44

**Comment by jacobweinstock** [comment]
FYI, https://github.com/tinkerbell/proposals/pull/44 has been merged. I will be putting out another PR to detail the responsibilities and privileges associated with the new roles.
